client/README.md

```diff
@@ -62,3 +62,32 @@ To generate a **class** component, run the following
 ```sh
 hygen component-class new --name NameOfComponent
 ```
+### CSS Class Names
+
+- To avoid style conflicts we want to make sure all our CSS selectors are unique.
+- This will be the naming convention, based on BEM. [https://css-tricks.com/bem-101/]
+```
+.ComponentName
+.ComponentName--modifier
+.ComponentName__elementName
+.ComponentName__elementName--modifier
```

Comment:
Did y'all decide on `elementName` over `element-name`? I think the PR I merged is using `element-name`

Reply:
Just a note. I personally like element-name and keeping the camel case for JS just to have things separate.